app/src/main/java/org/mozilla/tv/firefox/telemetry/TelemetryIntegration.kt

```diff
@@ -360,19 +360,19 @@ open class TelemetryIntegration protected constructor(
     }
 
     @AnyThread // pocketUniqueClickedVideoIDs is synchronized.
-    fun pocketVideoClickEvent(id: Int) {
+    fun pocketVideoClickEvent(id: String) {
```

Comment:
I don't love that we're removing type safety from the Pocket tile IDs but it seems low risk: the Pocket JSON is typed to Int, we change it to String to fit in a ChannelTile, and then it gets sent up directly as that String. I don't have any better solutions and I don't think it's worth spending too much time on so this is fine with me.